### PR TITLE
updated premailer documentation link

### PR DESCRIPTION
**Pull request by eveevans**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ premailer-rails is released under the MIT license. See the [license file].
 [actionmailer]: https://github.com/rails/rails/tree/master/actionmailer
 [nokogiri]:     https://github.com/sparklemotion/nokogiri
 
-[premailer documentation]: http://rubydoc.info/gems/premailer/1.7.3/Premailer:initialize
+[premailer documentation]: https://www.rubydoc.info/gems/premailer/1.10.2/Premailer:initialize
```
Comment (fphilipe) [inline README.md]: There is a URL without the version, which would be better: https://www.rubydoc.info/gems/premailer/Premailer:initialize

**Comment by eveevans** [comment]
Of course. 
But it should be on a  new pull request ? 
Or it can be changed from here ?

**Comment by fphilipe** [comment]
See this guide: https://gist.github.com/longtimeago/f7055aa4c3bba8a62197

Let me know if you need help 😉

**Comment by eveevans** [comment]
First time squashing 
Thanks for the tip and the patience for this small contribution @fphilipe

**Pull request merged**